terms/ontology.md

```diff
@@ -37,11 +37,19 @@ Example: OWL Ontology Language is based on Description Logics, OWL Ontology cons
 
 </details>
 
+{% hint style="warning" %}
+**Disambiguation**
+An ontology is technically equivalent to a schema. Both are formal specifications of some conceptualisation or abstract world model.
 
+From its usage in the Semantic Web community and the broader knowledge-based systems community, the term 'schema' is used to refer to simple conceptualisations, as might be written using less expressive knowledge representation languages such as RDF Schema (so making explicit the relationship between ontologies and database schemas). Conversely, the term 'ontology' is taken to describe more complex models, written in more expressive languages (OWL, DAML+OIL, Ontolingua, KIF, etc).
 
+There is a school of thought that considers ontologies to contain rule-based knowledge in addition to a relational characterisation, but this is far less prevalent in the SW community than elsewhere.
```

Comment:
You say in a comment this is taken from https://www.w3.org/wiki/SchemaVsOntology but haven't referred to that anywhere - please include a hyperlink to this source reference.

Reply:
I'm not sure what the best way to link this is. Should I put all the text in quotes?